gc/base/MemoryPool.hpp

```diff
@@ -422,6 +422,8 @@ class MM_MemoryPool : public MM_BaseVirtual
 	 * @return bytes of free memory in the pool released/decommited back to OS
 	 */
 	virtual uintptr_t releaseFreeMemoryPages(MM_EnvironmentBase* env);
+
+	virtual uintptr_t releaseFreeMemoryPages(MM_EnvironmentBase* env, uintptr_t includeMemoryType);
```

Comment:
MemoryPool does not need to be changed. Same for MPAOL subclass.
All free memory is always released. A higher entity decides if a pool should be released or not.